react/features/base/participants/components/ParticipantView.native.js

```diff
@@ -205,7 +212,7 @@ class ParticipantView extends Component<Props> {
         // black screen.
         const waitForVideoStarted = false;
```

Comment:
I think we should remove this, since we are here. There seems to be no way to make it work anyway.

Reply:
Sure thing! I did best effort here, please review more, as I'm not fully familiar with this approach.